guides/source/getting_started.md

```diff
@@ -2063,6 +2063,9 @@ Authentication challenge:
 
 ![Basic HTTP Authentication Challenge](images/getting_started/challenge.png)
 
+After entering the correct user name and password, you will remain authenticated
+until a different user name and password is required or the browser is closed.
+
```

Comment:
Our docs also generally use `username` and not `user name`, so I agree with this change.

Reply:
I was on the fence about this!  I also generally write "username".  I wrote "user name" here because the [challenge screenshot](https://github.com/rails/rails/blob/086935ea1db543037609cd33f3b4d29e9fdd5c9a/guides/assets/images/getting_started/challenge.png) displays "User Name", and because the parameter is `name`, as @p8 points out.  But let's go with the common spelling. :+1: